src/services/api/api.ts

```diff
@@ -61,11 +64,23 @@ export const api = createApi({
       transformResponse: (response: Bundle<ICondition>) =>
         transformBundleIntoFhirResource(response),
     }),
+    // Patient resources
+    apiPatientResourcesList: build.query<
```

Comment:
It's not juste a patient list anymore, it's _everything_ related to the patient.
I'd call it `apiPatientEverything`
add please add a link here to the docs:
https://www.hl7.org/fhir/operation-patient-everything.html